pom.xml

```diff
@@ -39,6 +39,11 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-webmvc</artifactId>
+			<version>5.2.6.RELEASE</version>
```

Comment:
general remark - I think we should extract versions of central frameworks into properties in the pom so that you have a better overview on them and so that you can change them in one placer if they have more than one artifact

Reply:
Second thought - maybe we should introduce a parent pom for our java projects, containing the version numbers of the frameworks throughout all our single projects?

Reply:
> Second thought - maybe we should introduce a parent pom for our java projects, containing the version numbers of the frameworks throughout all our single projects?

Definitely worth looking into but I don't think it is necessary now.